website_snippet_big_button/templates/snippets.xml

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2015-2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+     Copyright 2019 Tecnativa - Cristina Martin R.
+     License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl.html). -->
+
+<odoo>
+
+    <template id="s_big_button" name="Big Buttons">
+        <section class="oe_snippet_body readable big-buttons">
+            <div class="container">
+                <div class="row mt16 mb16">
+                    <div class="text-center col-6">
+                        <a href="/contactus" class="btn btn-success btn-lg font-weight-bold">
+                            <span>I want this</span>
+                            <i class="fa fa-thumbs-o-up fa-5x mt16 mb16"></i>
+                            <span>Request budget</span>
```

Comment:
Change for a `<p>` as it was before, and as it is in the ":-1:" button.

Reply:
Why? :thinking:

Reply:
because A is by default an inline element and some browsers do strange things. `inline-block` probably is more appropriated indeed.

Not to mention that semantically a link that includes a paragraph makes little sense and w/ ol' school html < 5 is not valid to have block elems inside a link, but this is another story :man_teacher:

Reply:
Well, this has been the structure used by this addon since 2015 and nobody notified a browser rendering problem, so I prefer to keep it as it was unless there's really a rendering problem. :thinking:  It seems that, although `<a>` should be inline, using it as a block is not really a problem in the Real World:tm:

Reply:
Let me rectify. I tested on runbot and, when changing the link, it strips out all block tags, so yes, we have to use spans or nothing.

Reply:
Fixed by adding `class="h3"`